core/src/main/resources/org/apache/spark/ui/static/timeline-view.css

```diff
@@ -222,12 +222,6 @@ rect.getting-result-time-proportion {
   stroke: #FF4D6D;
 }
 
-.vis-timeline .vis-item.executor.vis-selected {
```

Comment:
Is `.vis_timeline` still used anywhere?

Reply:
This means `.vis-item.executor.vis-selected`s which are descendants of `.vis-timeline` so I think it's not a problem.